.github/workflows/build-docker-image.yml

```diff
@@ -82,6 +79,18 @@ jobs:
           git config --global user.email "github-automation@users.noreply.github.com"
           git config --global user.name "Github Automation"
 
+      - name: Check Platform String
+        id: check
+        run: |
+          if [ ${{ inputs.multiarch }} == true ]; then
+            echo "Build Multiarch"
+            echo "::set-output name=platform::linux/amd64, linux/arm64"
```

Comment:
Rename output variable from `platform` to `platforms` to reflect that it may hold multiple items

Reply:
Done. However, the name on the input shall be defined as a single platform.